src/components/Navbar/SidebarNav.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from "react";
+import ReactDOM from "react-dom";
+import { NavLink } from "react-router-dom";
+
+import useWidth from "../../hooks/useWidth";
+
```

Comment:
could have added a picture of the change made it would have made it easier to review